heksher/clients/stub.py

```diff
@@ -91,7 +91,7 @@ def patch(self, setting: Setting[T], value: Union[T, Collection[Rule[T]]]) -> Co
             ]
             setting.update(self, first, rules)
         else:
-            setting.update(self, (), value, is_rule_collection=False)  # type: ignore
+            setting.update(self, (), value)  # type: ignore
```

Comment:
since you're sending un-colleted rules now, you need to send rules here as well
```suggestion
            setting.update(self, (), [((),value)])  # type: ignore
```